components/src/core/components/List/List.vue

```diff
@@ -335,7 +335,6 @@ export default defineComponent({
 
     const quickSearchKeywordSearch = () => {
       state.quickSearchTriggered = true;
-      quickSearchComponent.value.onBlur();
```

Comment:
`quickSearchComponent` ref is only used to trigger `onBlur()` in List.vue, since now it's not used please remove this ref and references to it from List.vue

Reply:
removed "quickSearchComponent"